requirements.txt

```diff
@@ -63,15 +66,18 @@ py==1.9.0
 pyasn1==0.4.8
 pycodestyle==2.6.0
 pycparser==2.20
+pycryptodomex==3.10.1
 pydot==1.4.1
 pyflakes==2.2.0
+pyjwkest==1.4.2
 PyJWT==2.0.1
 pyparsing==2.4.7
 pyrsistent==0.17.3
 pytest==6.1.2
 pytest-cov==2.10.1
 pytest-django==4.1.0
-pytest-factoryboy===2.1.0
+pytest-factoryboy==2.1.0
+pytest-mock==3.5.1
```

Comment:
I think we could use the unittest.mock(?)

Reply:
Yeah, this is actually probably unnecessary.